Formula/quantlib.rb

```diff
@@ -38,12 +37,20 @@ class Quantlib < Formula
   def install
     ENV.cxx11 if build.cxx11?
     if build.head?
-      Dir.chdir "QuantLib"
       system "./autogen.sh"
     end
+    if MacOS.version >= :mavericks
+      ENV.append "LDFLAGS", "-stdlib=libstdc++ -mmacosx-version-min=10.6"
+      ENV.append "CXXFLAGS", "-stdlib=libstdc++ -mmacosx-version-min=10.6"
```

Comment:
No matter what the linking problems are (or were), this is definitely not a solution but more of a dirty hack that is going to create new issues sooner than later. If the original issue still persists, it would be good to understand the root cause and fix it properly instead of perpetuating a more than two year old workaround …